<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelContentText</class>
 <widget class="QWidget" name="PanelContentText">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="SectionContainer" name="sectionText">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title" stdset="0">
      <string>Text</string>
     </property>
     <widget class="QWidget" name="layoutText">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="SectionContainer" name="sectionParagraph">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title" stdset="0">
      <string>Paragraph</string>
     </property>
     <widget class="QWidget" name="layoutParagraph">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="SectionContainer" name="sectionCharacter">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title" stdset="0">
      <string>Character</string>
     </property>
     <widget class="QWidget" name="layoutCharacter">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="SectionContainer" name="sectionLists">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title" stdset="0">
      <string>Lists</string>
     </property>
     <widget class="QWidget" name="layoutLists">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="SectionContainer" name="sectionTextOnPath">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title" stdset="0">
      <string>Text On Path</string>
     </property>
     <widget class="QWidget" name="layoutTextOnPath">
      <layout class="QVBoxLayout" name="verticalLayout_6"/>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SectionContainer</class>
   <extends>QWidget</extends>
   <header>section_container/section_container.h</header>
   <addpagemethod>addPage</addpagemethod>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
